packages/trip-details/src/TripDetails.story.tsx

```diff
@@ -271,3 +303,12 @@ export const TncTransitItineraryWithCustomMessagesAndDetails = makeStory(
 export const FlexItinerary = makeStory({
   itinerary: fareComponentsItinerary
 });
+
+const exampleItinerary = walkInterlinedTransitItinerary;
+export const FareDetailsComponent = () => (
+  <FareDetails
+    transitFares={exampleItinerary.fare}
```

Comment:
Use `englishFareKeyMap` and `frenchFareKeyMap` to demo localization (see line 165 in this file).